.github/workflows/nightly_ci.yaml

```diff
@@ -62,6 +62,8 @@ jobs:
         pip install .
         cd ..
         python -m pip install git+https://github.com/qutip/qutip-qip
+        python -m pip install jax jax[cpu] diffrax equinox
```

Comment:
These requirements for jax seem to already be in the setup.cfg. Is it still necessary to add them here?

Reply:
We could probably remove diffrax and equinox.
`jaxlib` (installed with `jax[cpu]`) is not a requirement of `jax` or `qutip-jax`, but is needed to run. 
We need to install it manually.

Reply:
https://github.com/google/jax/discussions/16380